tests/components/application-connector/Makefile

```diff
@@ -0,0 +1,45 @@
+NAMESPACE ?= "test"
+
+DOCKER_TAG ?= "PR-14743"
+DOCKER_PUSH_REPOSITORY ?= "eu.gcr.io/kyma-project"
+GOPATH ?= $(shell go env GOPATH)
+
+GATEWAY_TEST_IMAGE ?= "$(DOCKER_PUSH_REPOSITORY)$(DOCKER_PUSH_DIRECTORY)/gateway-test:$(DOCKER_TAG)"
+MOCK_APP_IMAGE ?= "$(DOCKER_PUSH_REPOSITORY)$(DOCKER_PUSH_DIRECTORY)/mock-app:$(DOCKER_TAG)"
+
+APP_URL = test-app.test.svc.cluster.local
+
+.PHONY: release test-os test image publish clean image-gateway publish-gateway test-gateway certs image-mock-app publish-mock-app
+
+test: test-gateway
```

Comment:
Maybe we can prepare it for other components. I mean some renaming from the default `test`, `image`, and other commands into `test-application-gateway`, `build-application-gateway`, and so on

Reply:
We have `image`, depending on `image-gateway`. IMO that's the good way to do it - subtasks for specific components, and a single task for easy running all